src/nickelmenu.cc

```diff
@@ -206,6 +207,15 @@ extern "C" __attribute__((visibility("default"))) MenuTextItem* _nm_menu_hook(vo
     return AbstractNickelMenuController_createMenuTextItem(_this, menu, label, checkable, checked, thingy);
 }
 
+QString nm_menu_pixmap(const char *path, const char *fallback) {
+    QPixmap a;
+    if (!a.load(QString(path))) {
```

Comment:
Umm, I could be very wrong, but does the QPixmap constructor need to be invoked here before loading data? (I know you're not actually using the image, but still).

Reply:
It's implicitly called in C++.